composer.json

```diff
@@ -6,9 +6,9 @@
     "license": "BSD-3-Clause",
     "require": {
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-cms": "4.x-dev",
-        "cwp/cwp-recipe-core": "2.x-dev",
-        "cwp/cwp": "2.x-dev",
+        "silverstripe/recipe-cms": "4.8.x-dev",
+        "cwp/cwp-recipe-core": "2.8.x-dev",
+        "cwp/cwp": "2.8.x-dev",
         "cwp/cwp-pdfexport": "1.x-dev",
         "silverstripe/html5": "2.x-dev"
```

Comment:
Should be 2.1.x-dev

See the commit at the rc1 tag https://github.com/silverstripe/cwp-recipe-cms/commit/c513f648a6b8aaab393b420df6bf804309559f1b